wormchain/proto/osmosis/tokenfactory/v1beta1/authorityMetadata.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package osmosis.tokenfactory.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/CosmosContracts/juno/x/tokenfactory/types";
```

Comment:
I think we'll need to change these references in every proto file to `"github.com/wormhole-foundation/wormchain/x/tokenfactory/types"`

Reply:
yes, thanks. I've updated those files, regenerated them and tested so we are now sourcing from our proto files.